### DISCUSSION TITLE
Social Accounts in Django Admin

### DISCUSSION THREAD

**Discussion by Dan1ell** [mediacms-io/mediacms#192] [Q&A] [answered]
Does this work?
If yes, how?
If no, what does it still need?

![social accounts in admin](https://user-images.githubusercontent.com/61538/119083059-a86fbd80-b9d5-11eb-93a6-d10549d1b8e1.png)

**Comment by mgogoulos** [comment, marked as answer]
It works yes, it's part of the django-allauth excellent package - https://django-allauth.readthedocs.io - that is responsible for lots of other goodies (eg making registration closed/open, allow self register/invite and many options, some of them are being in use in cms/settings.py. 

On django-allauth docs there are instructions on adding external authentication, supporting all major networks (google/twitter etc)
